src/app/DTT/event_button/index.js

```diff
@@ -18,6 +20,10 @@ const EventButton = ({ askData, onFinish, parentCode, sourceCode, config }) => {
 
   const { buttonBackgroundColor } = useProductColors()
 
+  if (!doubleBang(questionCode) && !isString(questionCode)) {
```

Comment:
What is the reason for doing !doubleBang(questionCode)? Isn't it the same as doing !questionCode?

Reply:
Corrected in latest commit